msteams-platform/m365-apps/actions-in-m365.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Actions in Microsoft 365
+description: In this article, learn more about the function of Actions and its use cases. 
+ms.date: 11/16/2023
+ms.author: mosdevdocs
+author: Vishnu
+ms.topic: Conceptual
+ms.subservice: m365apps
+---
+# Actions in Microsoft 365
+
+> [!NOTE]
+>
+> * Actions for Microsoft 365 is available in [public developer preview](../resources/dev-preview/developer-preview-intro.md).
+>
+> * Actions is supported for Microsoft 365 apps on web and desktop clients only and isn't supported on Outlook and Microsoft Teams.
+
+Actions enable your app to integrate seamlessly into the user's workflow and guide users to your app based on their intent and the content at hand, making task completion more efficient.
+Actions help you enhance your app's visibility and engagement with minimal development effort and improve user productivity by streamlining task completion and reducing the need to switch contexts across various Microsoft 365 apps. For instance, your app can act immediately on content files, expanding the range of interactions users can have with their content.
```

Comment:
This paragraph feels a bit verbose. Could we simplify and/or use similar wording from original intro paragraph of private preview docs:

Actions aim to integrate your app into your user's workflow by enabling easy discoverability and seamless interaction with their content. By directing users to your app with their intent and contextual content, Actions enable efficient task completion. This integration not only enhances the visibility and engagement of your app but also offers these benefits with minimal development effort.  